src/components/app/list/appList.modal.ts

```diff
@@ -61,7 +61,7 @@ const environmentModal = (env) => {
 }
 
 const getDefaultEnvironment = (envList): Environment => {
-    let env = envList.find((env) => env.default) || getLastDeployedEnv(envList) || envList[0]
+    let env = envList[0] || envList.find((env) => env.default) || getLastDeployedEnv(envList)
```

Comment:
it is same as let env = envList[0]
so if it is expected then please remove `|| envList.find((env) => env.default) || getLastDeployedEnv(envList)` or 
fix the condition